src/services/migration/MigrationFactory.php

```diff
@@ -21,11 +21,9 @@ public function __construct(Factory $factory, Environment $environment, Cli\Inpu
      */
     public function getMigrations(): array {
         return [
-            new HomeAuthXmlMigration($this->factory->getConfig(), $this->input),
             new HomePharsXmlMigration($this->factory->getConfig(), $this->input),
             new HomePhiveXmlMigration($this->factory->getConfig(), $this->input),
```

Comment:
Why do these two need `Cli\Input`? They should be run without user interaction, right?

Reply:
It was caused by a too a parent class the do too much. Fixed